src/rules/declaration-block-trailing-semicolon/README.md

```diff
@@ -10,6 +10,8 @@ a { background: orange; color: pink; }
 
 The trailing semicolon is the *last* semicolon in a declaration block and it is optional.
 
+This rule is not compatible with declaration blocks containing nested rules and at-rules.
```

Comment:
I think this might be a little more accurate: "This rule will ignore declaration blocks ending with nested rules and at-rules."

I think the problem is only when the nested block is at the end. And "not compatible" suggests things will break, but they will really just be ignored.